api/v1alpha1/spannerautoscaler_conversion_test.go

```diff
@@ -0,0 +1,331 @@
+package v1alpha1
+
+import (
+	"github.com/mercari/spanner-autoscaler/api/v1beta1"
+	"github.com/mercari/spanner-autoscaler/utils/pointer"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+	"time"
+)
+
+func TestSpannerAutoscaler_ConvertTo(t *testing.T) {
```

Comment:
Use [Ginkgo](https://onsi.github.io/ginkgo/) for writing tests. [Ref(https://book.kubebuilder.io/cronjob-tutorial/writing-tests.html)]

Just for a reference, you can take a look at the tests in `github.com/mercari/spanner-autoscaler/controllers` package (or also in the tests introduced in #65 in `api/v1beta1/spannerautoscaler_webhook_test.go`).